docs/metrics-events.md

```diff
@@ -221,7 +220,31 @@ Action name|Description
 `signup`|Create an account, i.e. `/account/create`
 `signin`|Sign in to an existing account, i.e. `/account/login`
 
-#### Connect method names ####
+#### Link names
+
+Possible values for `${linkName}` are:
+
+Link name|Description
+---------|-----------
+`app-store.android`|A Google Play Store link
+`app-store.ios`|An iOS App Store link
+`maybe_later`|'Maybe later' link
+`signin`|'Sign in' link
+`why`|'Why is this required?' link
+`create-account`|'Create an account' link
+`back`|'Back' link
+`support`|A SUMO link
+
+#### Experiment names
+
+Possible values for `${experiment}` are:
+
+Experiment name|Description
+---------------|-----------
+`connectAnotherDevice`|Connect another device, phase 1
+`sendSms`|Connect another device, phases 2 and 3
```

Comment:
My understanding is that the `sendSms` experiment now has 3 groups: `treatment`, `control` and `signinCodes`. There's no `signinCodes` experiment.

@shane-tomlinson am I on the right track or do I have the wrong end of the stick?